aspnetcore/client-side/index.md

```diff
@@ -3,14 +3,15 @@ title: Client-side development in ASP.NET Core
 author: rick-anderson
 description: 
 ms.author: riande
-ms.date: 08/09/2018
+ms.date: 08/22/2018
 uid: client-side/index
 ---
 # Client-side development in ASP.NET Core
 
 - [Use Gulp](xref:client-side/using-gulp)
 - [Use Grunt](xref:client-side/using-grunt)
 - [Use LibMan](xref:client-side/libman/index)
+  - [LibMan CLI](xref:client-side/libman/libman-cli)
```

Comment:
We settled on asterisks for lists. I'm cool either way, but I've been using asterisks in lists and converting dashes to asterisks on topic updates.